ajax/libs/reactive-elements/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "reactive-elements",
+  "version": "0.6.7",
+  "description": "Allows to use React.js component as custom element",
+  "filename": "reactive-elements.min.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/PixelsCommander/ReactiveElements.git"
+  },
+  "keywords": [
```

Comment:
I missed this, sorry!
In my opinion, the simple demo page can not be a homepage. The homepage should involve more complete information like documentation, library installation steps and so on.